tests/unit/services/user-idle-test.js

```diff
@@ -24,18 +24,17 @@ test('init sets up event listeners', function (assert) {
   service.get('userActivity').trigger(event);
 
   let stub = service.get('resetTimeout');
-  assert.ok(stub.calledOnce, 'resetTimeout was called');
+  assert.ok(stub.calledTwice, 'resetTimeout was called');
```

Comment:
Why would it be called twice?

Reply:
This is a revert to the original code where it was called twice :-)

Reply:
xD ok